src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.toString();
```

Comment:
`toString()` is common for all enum constants. It means, that for every constant `toString()` will be returning default, that’s not ok. So it’s better to use standard method of enum `name()` that will be returning always String representation of concrete enum constant.